src/data/zmod/basic.lean

```diff
@@ -103,9 +103,26 @@ instance has_repr : Π (n : ℕ), has_repr (zmod n)
 | 0     := int.has_repr
 | (n+1) := fin.has_repr _
 
-instance comm_ring : Π (n : ℕ), comm_ring (zmod n)
-| 0     := int.comm_ring
-| (n+1) := fin.comm_ring n
+instance comm_ring (n : ℕ) : comm_ring (zmod n) :=
```

Comment:
I think this could do with a `/- -/` comment (not visible in the docstring) explaining why this is necessary.